MapboxNavigation/RouteVoiceController.swift

```diff
@@ -369,7 +369,7 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
         }
         
         // Only localized languages will have a proper fallback voice
-        utterance.voice = AVSpeechSynthesisVoice(language: Bundle.main.preferredLocalizations.first)
+        utterance.voice = AVSpeechSynthesisVoice(language: Locale.preferredLanguages.first!)
```

Comment:
Instead of this change, I suspect [this line](https://github.com/mapbox/mapbox-navigation-ios/blob/98f9286234733bbee62f68c8fd899318772da2c8/MapboxNavigation/RouteVoiceController.swift#L273) that determines the Polly voice needs to be switched from Locale to Bundle.